src/main/java/seedu/address/ui/CalendarPanel.java

```diff
@@ -163,9 +163,11 @@ public void updateItem(LocalDate item, boolean empty) {
                             String finalTaskDate = "";
                             String taskDate = "";
                             try {
-                                Date deadline = ParserUtil.parseDate(task.getDeadline().date);
-                                DateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy");
-                                taskDate = dateFormat.format(deadline);
+                                if (!task.getDeadline().isEmpty()) {
```

Comment:
Hey @tpq95 I edited the code here because the calendar kept throwing errors when trying to load deadlines that were empty.